routemaster/state_machine/visualisation.py

```diff
@@ -0,0 +1,28 @@
+"""Visualisation code for state machines."""
+
+from typing import Dict, List, Union
+
+from routemaster.config import Action, StateMachine
+
+
+def nodes_for_cytoscape(
+    state_machine: StateMachine,
+) -> List[Dict[str, Union[Dict[str, str], str]]]:
```

Comment:
I think we should extract and name this type.

Reply:
Can then re-use it below too.

Reply:
4b9d8c0db1fd3b1f163467dae3d68656085b46d3